packages/core-extensions/content-containers/exam-edit/src/edit/index.vue

```diff
@@ -0,0 +1,173 @@
+            @click.stop="emit('delete')"
+          >
+            Delete Exam
+          </VBtn>
+        </VCol>
+      </VRow>
+    </VCard>
+    <VExpandTransition>
+      <div v-if="!collapsed" class="px-8 py-4">
+        <VAlert
+          v-if="!groups.length"
+          color="primary-darken-1"
+          icon="mdi-information-variant"
+          variant="tonal"
+          prominent
+        >
+          Click the button below to Create first question group.
+        </VAlert>
+        <VExpansionPanels v-model="expanded" rounded="lg" flat>
+          <AssessmentGroup
+            v-for="(group, index) in groups"
+            :key="group.uid"
+            :group="group"
+            :elements="elements"
+            :is-disabled="disabled"
+            :objectives="examObjectives"
+            :position="index"
+            :is-expanded="expanded === group.uid"
+            @save:element="$emit('save:element', $event)"
+            @update:element="$emit('update:element', $event)"
+            @reorder:element="$emit('reorder:element', $event)"
+            @delete:element="$emit('delete:element', $event)"
+            @update="$emit('update:subcontainer', $event)"
+            @delete="$emit('delete:subcontainer', group, 'group')" />
+        </VExpansionPanels>
+        <VBtn
+          v-if="!disabled"
+          :disabled="!container.id"
+          color="primary-lighten-5"
+          class="my-5"
+          @click.stop="createGroup">
+          <VIcon class="pr-2">mdi-folder-plus-outline</VIcon>
+          Add Question Group
+        </VBtn>
+      </div>
+    </VExpandTransition>
+  </VCard>
+</template>
+
+<script lang="ts" setup>
+import { activity as activityUtils, numberToLetter } from '@tailor-cms/utils';
+import { computed, ref, watch } from 'vue';
+import type { Activity } from '@tailor-cms/interfaces/activity';
+import type { ContentElement } from '@tailor-cms/interfaces/content-element';
+import type { ContentElementCategory } from '@tailor-cms/interfaces/schema';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import get from 'lodash/get';
+import last from 'lodash/last';
+import pluralize from 'pluralize';
+
+import AssessmentGroup from './AssessmentGroup.vue';
+
+interface Props {
+  name: string;
+  container: Activity;
+  elements: Record<string, ContentElement>;
+  position: number;
+  activities: Record<string, Activity>;
+  embedElementConfig?: ContentElementCategory[];
+  contentElementConfig?: ContentElementCategory[];
+  config?: Record<string, any>;
+  disabled?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  embedElementConfig: () => [],
+  contentElementConfig: () => [],
+  config: () => ({}),
+  disabled: false,
+});
+const emit = defineEmits([
+  'delete',
+  'add:subcontainer',
+  'save:element',
+  'update:element',
+  'reorder:element',
+  'delete:element',
+  'update:subcontainer',
+  'delete:subcontainer',
+]);
+
+const expanded = ref<string>();
```

Comment:
Consider adding context; what are expanded and collapsed used for (e.g. expandedAssessmentGroup)